core/src/main/resources/application.conf

```diff
@@ -362,7 +362,7 @@ database {
       driver = "slick.driver.HsqldbDriver$"
       db {
         driver = "org.hsqldb.jdbcDriver"
-        url = "jdbc:hsqldb:mem:${uniqueSchema};shutdown=false;hsqldb.tx=mvcc"
+        url = "jdbc:hsqldb:mem:${uniqueSchema};shutdown=false;hsqldb.tx=mvcc;allow_empty_batch=true"
```

Comment:
I guess not since the migration only applies to mysql. However there will be some flag required for mysql which I thought could be documented in the Changelog